backpack/extensions/secondorder/diag_hessian/activations.py

```diff
@@ -2,6 +2,11 @@
 from backpack.core.derivatives.sigmoid import SigmoidDerivatives
 from backpack.core.derivatives.tanh import TanhDerivatives
 from backpack.extensions.secondorder.diag_hessian.diag_h_base import DiagHBaseModule
+from backpack.core.derivatives.elu import ELUDerivatives
+from backpack.core.derivatives.selu import SELUDerivatives
+from backpack.core.derivatives.leakyrelu import LeakyReLUDerivatives
+from backpack.core.derivatives.logsigmoid import LogSigmoidDerivatives
+from backpack.extensions.secondorder.diag_ggn.diag_ggn_base import DiagGGNBaseModule
```

Comment:
I think that is not correct, you are using the `DiagGGNBaseModule` but it should be the `DiagHBaseModule`